torch/csrc/distributed/rpc/init.cpp

```diff
@@ -284,6 +286,46 @@ If the future completes with an error, an exception is thrown.
         return pyRpcPythonUdf(agent, dst, pickledPythonUDF, tensors);
       });
 
+  struct PythonFutureWrapper {
+    explicit PythonFutureWrapper(c10::intrusive_ptr<c10::ivalue::Future> fut)
+        : fut(std::move(fut)) {}
+
+    c10::intrusive_ptr<c10::ivalue::Future> fut;
+  };
+
+  py::class_<PythonFutureWrapper>(module, "PythonFutureWrapper")
```

Comment:
this should be bind to "Future" instead of "PythonFutureWrapper" right?

Reply:
right, good catch